wolox-books/src/app/screens/register/register.component.scss

```diff
@@ -1,101 +1,7 @@
-@import url(https://fonts.google.com/specimen/Roboto?selection.family=Roboto:300,400,500,700,900);
-@import "src/scss/application.scss";
+@import "./src/app/app.component.scss";
```

Comment:
What's app.component.scss for?

Reply:
This file is were the common classes are define, for login and sign-up form.